lib/healthlocker/slam/carer_connection.ex

```diff
@@ -20,6 +20,7 @@ defmodule Healthlocker.Slam.CarerConnection do
   end
 
   defp validate_slam(changeset, options \\ []) do
+    IO.inspect changeset
```

Comment:
Not sure if you want to keep this log of the changeset

Reply:
Whoops, no I do not. I'll remove that.